src/modules/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,8 @@
+export class CreateUserDto {
```

Comment:
it will be easier to make one file for use dto's

Reply:
not critical

Reply:
created one file for use dto's